src/views/splash/hoc/top-banner.jsx

```diff
@@ -0,0 +1,108 @@
+    description: 'ttt.AnimateYourNameDescription',
+    thumbUrl: '/images/ttt/animate-your-name.jpg',
+    bannerUrl: '/images/ttt/animate-your-name-banner.jpg',
+    tutorialLoc: '/projects/editor/?tip_bar=name',
+    activityLoc: 'cards.nameCardsLink',
+    guideLoc: 'guides.NameGuideLink'
+};
+
+var TopBanner = injectIntl(React.createClass({
+    type: 'TopBanner',
+    getInitialState: function () {
+        // use translated tile
+        var formatMessage = this.props.intl.formatMessage;
+        var translatedTile = {};
+        translatedTile = {
+            title: formatMessage({id: nameTile.title}),
+            description: formatMessage({id: nameTile.description}),
+            tutorialLoc: nameTile.tutorialLoc,
+            activityLoc: formatMessage({id: nameTile.activityLoc}),
+            guideLoc: formatMessage({id: nameTile.guideLoc}),
+            thumbUrl: nameTile.thumbUrl,
+            bannerUrl: nameTile.bannerUrl
+        };
+        return {currentTile: translatedTile, TTTModalOpen: false};
+    },
+    showTTTModal: function () {
+        this.setState({TTTModalOpen: true});
+    },
+    hideTTTModal: function () {
+        this.setState({TTTModalOpen: false});
+    },
+    render: function () {
+        return (
+            <TitleBanner className="mod-splash-top">
+                <FlexRow className="banner-top inner">
+                    <FlexRow className="top-animation">
+                        <img
+                            src="/images/hoc/s.png"
+                            alt="C"
+                            className="top-animation-letter mod-letter-s"
+                        />
+                        <img
+                            src="/images/hoc/c1.png"
+                            alt="C"
+                            className="top-animation-letter mod-letter-c1"
+                        />
+                        <img
+                            src="/images/hoc/r.png"
+                            alt="R"
+                            className="top-animation-letter mod-letter-r"
+                        />
+                        <img
+                            src="/images/hoc/a.png"
+                            alt="A"
+                            className="top-animation-letter mod-letter-a"
+                        />
+                        <img
+                            src="/images/hoc/t.png"
+                            alt="T"
+                            className="top-animation-letter mod-letter-t"
+                        />
+                        <img
+                            src="/images/hoc/c2.png"
+                            alt="C"
+                            className="top-animation-letter mod-letter-c2"
+                        />
+                        <img
+                            src="/images/hoc/h.png"
+                            alt="H"
+                            className="top-animation-letter mod-letter-h"
+                        />
```

Comment:
@carljbowman @chrisgarrity I think the letters are showing up a little bit pixel-y around the edges when I view on a browser:
<img width="722" alt="screen shot 2017-11-07 at 4 59 08 pm" src="https://user-images.githubusercontent.com/1815772/32520887-99264fa0-c3df-11e7-9f7d-6436b4a0cd90.png">

Reply:
@carljbowman do you want to update the letter images?

Reply:
From @carljbowman The letters are intended to have textured edges, and the texture might just be right on the edge of appearing pixelated and texture on smaller screens. We'll leave them as is for now, but keep it in mind for the future.

Reply:
@chrisgarrity @carljbowman awesome thanks!